docs/documentation/getting-started/load.mdx

```diff
@@ -2,19 +2,30 @@
 title: Load Data from Postgres
 ---
 
-The easiest way to copy data from another Postgres into ParadeDB is with the `pg_dump` and `pg_restore` utilities. These are installed by default when you install `psql`.
+The easiest way to copy data from another Postgres into ParadeDB is with the `pg_dump` and `pg_restore` utilities. These are
+installed by default when you install `psql`.
 
-This approach is ideal for quickly bootstrapping your ParadeDB cluster with some data. To keep ParadeDB in sync with an existing Postgres database in real time, [logical replication](/deploy/replication) should be configured.
+This approach is ideal for quickly bootstrapping a development ParadeDB cluster
+with a snapshot of your data. To deploy ParadeDB in production, kept in sync with an existing Postgres database in real
+time, [logical replication](/deploy/replication) should be configured instead.
+
+<Note>
+  If your primary Postgres is in a virtual private cloud (VPC), we recommend
+  deploying ParadeDB on a compute instance within your VPC to avoid exposing
+  public IP addresses and needing to provision traffic routing rules. For the
+  best performance, we recommend using a storage-optimized instance.
```

Comment:
The line about using a storage-optimized instance is out of place for a "getting started load data" guide

Reply:
Removed